### PR TITLE
WORK: Moving azure-sdk-for-net-pr NetworkManagement2 project source from ignite into public branch

### PR DESCRIPTION
**Pull request by Nilambari**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Nilambari__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilamd). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by Nilambari** [comment]
@DeepakRajendranMsft Can you Please do the review? Thanks!

File: src/NetworkManagement2/packages.config [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.12.0-preview" targetFramework="net40" />
```
Comment (DeepakRajendranMsft) [inline src/NetworkManagement2/packages.config]: @Nilambari We should not be taking dependency on Resources. I check in -pr Build branch and i dont find it there. it should not required for ignite as well. Can we remove this dependency please?

Comment (Nilambari) [inline src/NetworkManagement2/packages.config]: Done. Thanks!

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@Nilambari - loo9ks like an issue with the nuspec file?

**Comment by yugangw-msft** [comment]
My change coves this

**Pull request closed without merging**